plugins/modules/sda_fabric_virtual_networks_workflow_manager.py

```diff
@@ -987,26 +992,32 @@ def create_fabric_vlan(self, vlan_payloads):
             class status accordingly.
         """
 
-        try:
-            payload = {"payload": vlan_payloads}
-            task_name = "add_layer2_virtual_networks"
-            task_id = self.get_taskid_post_api_call("sda", task_name, payload)
+        req_limit = self.params.get('sda_fabric_vlan_limit', 50)
```

Comment:
sda_fabric_vlan_limit .. is it a new variable? Do we document this?

Reply:
yes, it's already documented.